encrypted-media/scripts/playback-temporary-expired.js

```diff
@@ -44,16 +44,18 @@ function runTest(config,qualifier) {
 
             assert_in_array(event.messageType, ['license-request', 'individualization-request']);
 
-            var expiration = Date.now().valueOf();
-            config.messagehandler(event.messageType, event.message, expiration).then(function(response) {
-                return event.target.update(response);
-            }).then(test.step_func(function(){
-                assert_approx_equals(event.target.expiration, expiration, 2000, "expiration attribute should equal provided expiration time");
-                test.step_timeout(function() {
-                    _video.play();
-                    test.step_timeout(function() { test.done(); }, 2000);
-                }, 5000);
-            })).catch(onFailure);
+            var expiration = Date.now().valueOf() - 60000;
```

Comment:
I've been testing this and am having my licence requests rejected if I issue them with an already expired time. If I alter this time to be in the future, but such that the licence expires by the time that wait below is done, then I get expected behaviour. However, then we have a race on the network: e.g. if the network is slow on the request reaching the licence server it may reject the request.